LiteLoader/Header/MC/Types.hpp

```diff
@@ -713,6 +713,26 @@ enum class ContainerType : char {
     SMITHING_TABLE         = 33,
 };
 
+enum class GameType : int {
+    Survival  = 0,
+    Creative  = 1,
+    Adventure = 2
+};
+
+enum class Difficulty : int {
+    Peaceful = 0,
+    Easy     = 1,
+    Normal   = 2,
+    Hard     = 3
+};
+
+enum class PlayerPermissionLevel : int {
```

Comment:
enum PlayerPermissionLevel shoule be `enum` instead of `enum class` 
ref: https://github.com/LiteLDev/LiteLoaderBDS/blob/e601f387cc6c6c58fc4382d90e74fcc16af6073c/LiteLoader/Header/MC/PermissionsHandler.hpp#L70

Reply:
> enum PlayerPermissionLevel shoule be `enum` instead of `enum class` ref:

**Fixed!**